lang/nl/lang.php

```diff
@@ -86,6 +86,7 @@
         'layout' => 'Layout',
         'description' => 'Beschrijving',
         'preview' => 'Voorbeeld',
+        'duplicate' => 'Duplicaat',
```

Comment:
Something minor;
In Dutch, the verb 'To duplicate' is 'Dupliceren'
'Duplicaat' is a noun, a duplicated item

This is probably the case for the other languages as well
I guess you used google translate 'Duplicate' but you should've used 'To duplicate' instead

Reply:
I've actually translated from german "duplizieren" wich is a verb. I will fix it and remove for all languages I do not speak.